### PR TITLE
Adding scanoss

### PR DESCRIPTION
**Pull request by o-konoval**
Adding scanoss in build.yml for testing purposes

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.
- [x ] CI system update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Below is just an example and you can delete it when you contribute.

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -32,3 +32,11 @@ jobs:
...
     - name: Build with Maven
       run: mvn -B package --file pom.xml
+      
+    - uses: actions/checkout@v4
+    - name: Install Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'   
+    - name: Install scanoss-py
+      run: pip3 install scanoss
```
Comment (o-kopysov) [inline .github/workflows/build.yml:42]: scanoss should be installed before tests execution

**Review by o-kopysov — CHANGES_REQUESTED** [review]
1. We have to replace v4 with hash
2. Target branch is o-kopysov:ut-add-more

File: .github/workflows/build.yml (hunk 1 above)
Comment (o-konoval) [reply .github/workflows/build.yml]: Yes fixed

**Comment by o-konoval** [comment]
Please re-review changes applied.

**Comment by o-kopysov** [comment]
I included all these changes into PR #299 .
Thank you!

**Pull request closed without merging**